README.md

```diff
@@ -33,3 +33,4 @@ Check out these examples:
 - [Honeycomb](./packages/honeycomb)
 - [Sentry](./packages/sentry)
 - [Stytch](./packages/stytch)
+- [Early hints](./packages/early-hints/)
```

Comment:
Can you bump it up to between "Cloudflare Access" and "Google Chat", just so its in alphabetical order?

```suggestion
- [Early Hints](./packages/early-hints/)
```